project/plugins.sbt

```diff
@@ -1,6 +1,9 @@
-addSbtPlugin("ch.epfl.scala"                     % "sbt-bloop"        % "1.4.0-RC1")
-addSbtPlugin("io.github.davidgregory084"         % "sbt-tpolecat"     % "0.1.10")
-addSbtPlugin("com.alejandrohdezma"               % "sbt-fix"          % "0.2.6")
-addSbtPlugin("com.thoughtworks.sbt-api-mappings" % "sbt-api-mappings" % "3.0.0")
-addSbtPlugin("de.heikoseeberger"                 % "sbt-header"       % "5.4.0")
-addSbtPlugin("org.scalameta"                     % "sbt-mdoc"         % "2.1.1")
+addSbtPlugin("com.alejandrohdezma"               % "sbt-fix"           % "0.2.6")
+addSbtPlugin("com.alejandrohdezma"               % "sbt-github-mdoc"   % "0.5.2")
+addSbtPlugin("com.alejandrohdezma"               % "sbt-github-header" % "0.5.2")
+addSbtPlugin("com.alejandrohdezma"               % "sbt-mdoc-toc"      % "0.1")
```

Comment:
Do you know how these mdoc plugins relate to sbt-microsites?

Reply:
I haven't tested it with `sbt-microsites` yet but is on my to-do list, although there shouldn't be any problem since they are just using standard mdoc variables and extensions.